kafka-base/Dockerfile

```diff
@@ -10,20 +10,20 @@ RUN groupadd -r -g 1001 kafka && useradd -r -m -u 1001 -g kafka kafka
 
 # Set Scala and Kafka version
 ENV SCALA_VERSION=2.11
-ENV KAFKA_VERSION=0.11.0.1
+ENV KAFKA_VERSION=1.0.0
 
 # Set Kafka MD5checksum
```

Comment:
I think we changed to sha512 before this PR, but we should fix this comment.

Reply:
Yeah, we changed the checksum before. I already changed the filename in this PR, but it seems I didn't noticed the comment. Should be fixed now.